transport/grpc/outbound_test.go

```diff
@@ -200,3 +202,84 @@ func TestCallStreamWithInvalidPeer(t *testing.T) {
 		}.Error(),
 	)
 }
+
+func TestCallSvcMatch(t *testing.T) {
```

Comment:
Same as the other PR: Call this Service, not Svc

Reply:
resolved.